### ISSUE TITLE
Kimura2Paramter to ks value/Time

### ISSUE THREAD

**Issue by manoharbisht1998** [oushujun/LTR_retriever#170]
Hi 
I have repeat landscape with Kimura2Parameter on x axis. I want to check the time of the burst of LTRs by using that repeat landscape. is there a way to convert the Kimura2Parameter values to time or ks value?

Thanks

**Comment by oushujun** [comment]
T = K/2u, where K is divergence which can be estimated using the K2P model. The default is the JC69 model. u is mutation rate (per bp per year).

**Comment by manoharbisht1998** [comment]
Thank you for the answer. However, I have the repeat landscape (attached below)
![Screenshot (70)](https://github.com/oushujun/LTR_retriever/assets/138770031/b72d8050-24e1-434e-864d-3d01037b7cf8)
I want to calculate the insertion time. According to formula T =K/2u, isn't it similar to Ks time estimation; viz T=Ks/2u? Can we say that K2P value is equal to Ks?

**Comment by oushujun** [comment]
Yes, K2P is one of the popular ways to estimate K. Be aware that your K may be the estimated divergence from the given library sequences that may not represent the ancestral sequence of the TE family, thus the calculated T may be not equal to the insertion time of that TE sequence.

**Comment by oushujun** [comment]
@manoharbisht1998 Can you please let me know the method you used to calculate the K2P distance? I want to add an option to give K2P values for intact LTR elements in the output.

**Comment by oushujun** [comment]
K2P model is added and becomes the default model for identity and age estimation in LTR_retriever since v3.0.2.
